docs/build-parachains-rococo.md

```diff
@@ -202,3 +202,67 @@ _Update: 15. Jan. 2021_
 If you would like to test your setup first on a local machine, you should be able to do so by
 following the instructions in the readme
 [launch a local setup](https://github.com/paritytech/cumulus#launch-a-local-setup-including-a-relay-chain-and-a-parachain).
+
+# Chachacha V1 - The pre-rococo environment
+
+Chachacha is a Rococo based network configured and supported by Centrifuge.
+
+The purpose of Chachacha is to serve as a support network to ease and speed up the onboarding of new
+parachains in Rococo.
+
+It helps:
+
+- To get the parachains in the waiting list to a state that is ready to be added to Rococo for
+  further performance and stability testing
+- Parachains to find issues beforehand, by integrating earlier.
+
+## Characteristics
+
+- Chachacha will be at par with the latest Rococo polkadot/cumulus/substrate version
+- The network will be refreshed and restarted frequently
+- Inclusion process is analogous to Rococo's
+- Notifications and Support will be given in the Rococo and Rococo Validator Lounge Element channels:
+    - Tag your question using the following user handle: @mikiquantum:matrix.org
```

Comment:
Please don't bootstrap on our rococo channel. I would much rather have a dedicated one.

Reply:
it's fine to have people be in both, but you have a separate network here, just happens to be based on the same codebase :)